src/matteruser.js

```diff
@@ -366,9 +366,6 @@ class Matteruser extends Adapter {
 
     let text = mmPost.message;
     if (msg.data.channel_type === 'D') {
-      if (!new RegExp(`^@?${this.robot.name}`, 'i').test(text)) { // Direct message
```

Comment:
It could be good idea to create the regex just one time at initialization instead of each time ?

Is it possible to avoid RegExp ? using startWith ?

Reply:
The purpose of this PR is to remove this block as it sounds not necessary. Do you know any use case where we need to keep this ?

Reply:
mouarf sorry I'm not awake :)